app/qml/editor/inputrelation.qml

```diff
@@ -279,13 +281,20 @@ Item {
 
       pageTitle: qsTr( "Linked features" )
       allowSearch: false //TODO search
+      layerHasGeometry: false
+      toolbarVisible: !root.parent.readOnly
+      focus: true
```

Comment:
Why does it need to be set to true?

Reply:
Just fixed the focus in forms (for back button). I am not 100% sure, but this should catch focus when other form is being closed.